CSC8503/SuspicionSystem/GlobalSuspicionMetre.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include "SuspicionMetre.h"
+#include <vector>
+
+const int DT_UNTIL_GlOBAL_RECOVERY = 8;
+class GlobalSuspicionMetre :
+    public SuspicionMetre
+{
+public:
+    const enum instantGlobalSusCause
+    {
+        allarmTriggered, flagCaptured
+    };
+
+    const enum activeGlobalSusCause
+    {
+        passiveRecovery
+    };
+
+    void AddInstantGlobalSusCause(instantGlobalSusCause inCause);
+
+    bool AddActiveGlobalSusCause(activeGlobalSusCause inCause);
```

Comment:
These should all be bool, or all be void. 

I'm in two minds about the bool names - I've seen this pattern before (adding an entity successfully returns true), but it does break our rule about the method name being answerable as 'true' or 'false'. I'm leaning towards it being fine, but I'll put it to the rest of the team.